core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -2952,8 +2955,14 @@ private void freeSymbolMapWriters() {
         }
     }
 
-    private long getO3RowCount0() {
-        return (masterRef - o3MasterRef + 1) / 2;
+    private void copyVersionAndLagValues() {
```

Comment:
wrong place for method, methods are not sorted in the file

Reply:
formatted code here: https://github.com/questdb/questdb/pull/2759/commits/860132cab511b6cdc911048dd8d269d649ba7315